File: 35-Search Insert Position/sol2.cpp
class Solution {
public:
    int searchInsert(vector<int>& nums, int target) {
        // if(target <= nums[0]){
        //     return 0;
        // }
        for(int i=0; i<nums.size();i++){
            // cout<<i;
            // if(nums[i] == target){
            //     return i;
            // }
            // if(target> nums[i] && target < nums[i+1] && target != nums[i+1] && target != nums[i] ){
            //     return i+1;
            // }
            if(target == nums[i] || target< nums[i]) return i;
        }
        return nums.size();
    }
};

File: 1822-Sign of the Product of an Array/sol2.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
      if 0 in nums:
          return 0
          
      count =0
      for i in nums:
          if i<0:
              count += 1
    
      return 1 if count %2 ==0 else -1
        
